pkg/mcs/resource_manager/server/resource_group.go

```diff
@@ -163,17 +113,13 @@ func FromProtoResourceGroup(group *rmpb.ResourceGroup) *ResourceGroup {
 	}
 	switch group.GetMode() {
 	case rmpb.GroupMode_RUMode:
-		if settings := group.GetRUSettings(); settings != nil {
-			rg.RUSettings = NewRequestUnitSettings(settings.GetRU())
+		if group.GetRUSettings() == nil {
+			rg.RUSettings = NewRequestUnitSettings(nil)
```

Comment:
I found CheckAndInit has redundant code and function with FromProtoResourceGroup,
So i replace `CheckAndInit` with `FromProtoResourceGroup`.
- for `CheckAndInit` can keep original logic
- for other way which call `FromProtoResourceGroup`, if set nil, just keep it.
And `RequestRU` will check it.
So i think it will be ok.